src/assets/locales/ko.json

```diff
@@ -0,0 +1,1143 @@
+    },
+    "disableAnimations": {
+      ".": "애니메이션 효과 비활성화",
+      "hint": "성능 향상을 위해 모든 애니메이션 효과를 비활성화합니다."
+    },
+    "disableColumns": {
+      ".": "가로열 확장식 배치 비활성화",
+      "hint": "확대/축소 및 남은 공간에 관계없이 가로 2열로의 확장식 배치를 완전히 비활성화합니다."
+    },
+    "disableStandees": {
+      ".": "몬스터 말 비활성화",
+      "hint": "몬스터 말을 보여주지 않습니다(능력치 숨기기 설정 시, 일반/정예 몬스터 별도 추가)."
+    },
+    "dragValues": {
+      ".": "주도권/체력/경험치/노획 드래그",
+      "hint": "값 변경을 위해 드래그 방색을 지원합니다. 주도권을 변경하려면 캐릭터 초상화에서 드래그하십시오. 체력/경험치/노획 부분의 경우는 현재 값 위에서 드래그하십시오."
+    },
+    "eliteFirst": {
+      ".": "정예 몬스터 말 우선",
+      "hint": "몬스터 말을 정렬할 때, 정예 몬스터 말을 우선으로 보여줍니다."
+    },
+    "expireConditions": {
+      ".": "자동으로 상태 처리",
+      "hint": "예를 들어서 강화 상태가 차례 이후에 제거되는 것처럼, 특정 상태를 대상의 차례 이후에 자동으로 제거합니다."
+    },
+    "feedback": {
+      ".": "피드백 버튼 비활성화",
+      "hint": ""
+    },
+    "fhStyle": {
+      ".": "프로스트헤이븐 스타일",
+      "hint": "프로스트헤이븐 스타일로 변경합니다."
+    },
+    "fontsize": {
+      ".": "능력 카드의 글씨 크기",
+      "dec": "능력 카드의 글씨 크기 줄이기",
+      "hint": "능력 카드 및 능력치 항목의 글씨 크기 설정하기",
+      "inc": "능력 카드의 글씨 크기 키우기",
+      "reset": "능력 카드의 글씨 크기 초기화"
+    },
+    "fullscreen": {
+      ".": "전체 화면",
+      "hint": "전체 화면으로 전환"
+    },
+    "ge5Player": {
+      ".": "5인 이상 규칙",
+      "hint": ""
+    },
+    "helperDefaults": {
+      ".": "도우미 설정을 기본값으로 적용하기",
+      "hint": ""
+    },
+    "hideAbsent": {
+      ".": "부재 중인 대상 숨기기",
+      "hint": "부재 중인 대상을 목록에서 보이지 않게 합니다."
+    },
+    "hideStats": {
+      ".": "능력치 숨기기",
+      "hint": "출현하지 않은 몬스터의 능력치 항목을 숨깁니다."
+    },
+    "hints": {
+      ".": "힌트 보기",
+      "hint": "메인 메뉴 옆의 공간에 바로 가기와 힌트를 표시합니다."
+    },
+    "initiativeRequired": {
+      ".": "주도권 필요",
+      "hint": "비활성화 시, 각 항목의 순서는 수동으로 설정해야 합니다."
+    },
+    "levelAdjustment": {
+      ".": "시나리오 레벨 조정",
+      "hint": ""
+    },
+    "levelCalculation": {
+      ".": "자동 시나리오 레벨",
+      "hint": ""
+    },
+    "locale": {
+      ".": "언어",
+      "de": "독일어 (Deutsch)",
+      "en": "영어 (English)",
+      "fr": "프랑스어 (Français)",
+      "ko": "한국어"
+    },
+    "maxUndo": {
+      ".": "실행 취소를 할 수 있는 최대 값",
+      "hint": ""
+    },
+    "moveElements": {
+      ".": "원소 상태 자동으로 갱신",
+      "hint": "각 라운드 전에 원소 상태를 자동으로 변경합니다."
+    },
+    "randomStandees": {
+      ".": "무작위 몬스터 말",
+      "hint": "사용하지 않은 무작위 번호의 몬스터 말을 추가합니다."
+    },
+    "scenarioNumberInput": {
+      ".": "시나리오 번호 입력",
+      "hint": ""
+    },
+    "scenarioRules": {
+      ".": "시나리오 규칙 보기",
+      "hint": "시나리오 특별 규칙을 팝업창으로 보여줍니다."
+    },
+    "theme": {
+      ".": "테마",
+      "default": "기본값",
+      "fh": "프로스트헤이븐",
+      "hint": "배경과 막대 및 글씨 유형을 선택합니다.",
+      "modern": "모던 (%wip%)"
+    },
+    "zoom": {
+      ".": "확대/축소",
+      "in": "확대",
+      "out": "축소",
+      "reset": "확대/축소 초기화"
+    }
+  },
+  "spoiler": {
+    "add": "스포일러 항목 추가",
+    "all": "모든 스포일러 항목 공개하기",
+    "placeholder": "???"
+  },
+  "state": {
```

Comment:
Where can I find this "state" entry and its sub-entries?

Reply:
Mean this https://github.com/Lurkars/gloomhavensecretariat/blob/main/src/assets/locales/en.json#L893?

Reply:
Yes, that's right.

I wonder where the [corresponding item](https://github.com/Lurkars/gloomhavensecretariat/blob/main/src/assets/locales/en.json#L893) is being displayed on the website.

[https://gloomhaven-secretary.de/](https://gloomhaven-secretary.de/)

Reply:
Ah okay. That are the texts for undo/redo! So those appear in main menu after taken corresponding actions.

Reply:
i got it.

thanks : )